actors/runtime/src/main/java/cloud/orbit/actors/concurrent/WaitFreeExecutionSerializer.java

```diff
@@ -74,7 +74,7 @@ public WaitFreeExecutionSerializer(final ExecutorService executorService, Object
         final Task<R> completion = new Task<>();
 
         int queueSize = size.get();
-        if (DEBUG_ENABLED && queueSize >= maxQueueSize / 10)
+        if (DEBUG_ENABLED && queueSize >= maxQueueSize / 10 && queueSize % 1000 == 0)
```

Comment:
As we discussed, should be maybe go with
```
if (DEBUG_ENABLED && (queueSize >= maxQueueSize / 10 && queueSize % (maxQueueSize / 10) == 0))
```

It would make the logs show the progression of how close we're getting, and scale for any queue size.